cc/algorithms/algorithm.h

```diff
@@ -177,6 +182,7 @@ class Algorithm {
   static constexpr double kFullPrivacyBudget = 1.0;
 
   const double epsilon_;
+  const double delta_;
```

Comment:
Where is `delta_`/`GetDelta` used?

Reply:
We use delta_ for the constructor. Also I think we need getDelta in the future for budget splitting and bounded algorithms

Reply:
Yes, this makes sense.  Typically we would only add code when it is required.  But since this will be required for the next PR when adding Gaussian to BoundedSum, we can keep it as is.